playerctl/playerctl-player.c

```diff
@@ -249,14 +249,14 @@ static void playerctl_player_class_init (PlayerctlPlayerClass *klass) {
         "Player name",
         "The name of the player mpris player",
         NULL, /* default */
-        G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY);
+        G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY| G_PARAM_STATIC_STRINGS);
```

Comment:
Yes, it tells GLib that *"the string used as name when constructing the parameter is guaranteed to remain valid and unmodified for the lifetime of the parameter."* (ee [here](https://developer.gnome.org/gobject/unstable/gobject-GParamSpec.html#G-PARAM-STATIC-STRINGS:CAPS)). This allows for an optimization where GLib does not have to duplicate the string (and free it) for each instantiation of `PlayerctlPlayer`, because it now knows that the string remains constant.